db/models/Entry.js

```diff
@@ -0,0 +1,39 @@
+import mongoose from "mongoose";
+
+const { Schema } = mongoose;
+
+const entrySchema = new Schema(
+  {
+    showAddButton: { type: Boolean, required: true },
+    isDictionaryEntry: { type: Boolean, required: true },
+    slug: { type: String, required: true },
+    japanese: {
+      type: Object,
+      required: true,
+      properties: {
+        word: { type: String },
+        reading: { type: String },
+      },
+    },
+    english: { type: Array, required: true },
```

Comment:
You could type this field as an array of strings (if they are strings)

Reply:
Yes, they are strings, but I don't know how many.
Would it be 
`[{ type: String }, { type: String },{ type: String }]`
? 
 (I looked it up in the mongodb documentation and it said to use what I put before, but I am happy to change it :) )

Reply:
I think it should be: 
``english: { type: [String], required: true}``